subsys/spm/spm.c

```diff
@@ -178,6 +180,53 @@ extern int irq_target_state_is_secure(unsigned int irq);
 	((SPU_PERIPHID_PERM_LOCK_Locked << SPU_PERIPHID_PERM_LOCK_Pos) &       \
 	 SPU_PERIPHID_PERM_LOCK_Msk)
 
+#define FLASH_NSC_REGION(reg_number)                                           \
```

Comment:
Perhaps have this in some spm/spu helper header file?

Reply:
Sure, should I move all the existing defines as well?

Reply:
I would say so, as long as they seem to belong there.

Reply:
Done